test/tests/notebooks/install_location_install.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
Could you add a test here that runs this notebook: https://github.com/google/swift-jupyter/blob/master/test/tests/simple_notebook_tests.py ?

The test could run the notebook twice and assert that no compilation happens the second time (which is what should happen if you do my top-level comment).